gwent.js

```diff
@@ -1687,35 +1928,90 @@ class UI {
 		this.preview = document.getElementsByClassName("card-preview")[0];
 		this.previewCard = null;
 		this.lastRow = null;
-		document.getElementById("pass-button").addEventListener("click", () => player_me.passRound(), false);
+		if (!navigator.userAgentData.mobile) {
```

Comment:
This breaks compatibility with Firefox. Something like `if (!("ontouchstart" in document.documentElement))` solves this problem